src/dashboardgeneral.html

```diff
@@ -62,7 +62,7 @@ <h2>${HeaderBranding}</h2>
                         <input is="emby-input" type="text" id="txtLoginDisclaimer" label="${LabelLoginDisclaimer}" />
                         <div class="fieldDescription">${LabelLoginDisclaimerHelp}</div>
                     </div>
-                    <div class="inputContainer">
+                    <div class="inputContainer customCSS">
```

Comment:
```suggestion
                    <div class="inputContainer customCssContainer">
```

It's slightly clearer this way.

Reply:
Thanks, I'll change it